src/bin/cli/mod.rs

```diff
@@ -142,63 +288,147 @@ pub struct Package {
     pub import_map: Option<Vec<(String, PathBuf)>>,
 }
 
-#[derive(Args)]
+#[derive(Args, Deserialize, Debug, PartialEq)]
+pub struct CompilePackage {
+    #[arg(name = "INPUT", help = "Solidity input files",value_parser = ValueParser::path_buf(), num_args = 1.., required_unless_present = "CONFFILE")]
```

Comment:
Same as my comment above. Then, given a `solang.toml` file is present, we can just do `solang compile` (akin to how you can just `cargo build` in Rust projects). Which I'd find really neat

Reply:
Good pointer, Thanks